setup.py

```diff
@@ -18,7 +17,7 @@ def _get_version():
     curr_dir = os.path.dirname(os.path.abspath(__file__))
     with open(os.path.join(curr_dir, 'smart_open', 'version.py')) as fin:
         #
-        # __version__ = '1.8.4'
+        # example: __version__ = '1.8.4'
```

Comment:
?

Reply:
Contents of the version.py for reference